libs/design-system/buttons/radio-button/src/lib/global-styles.component.scss

```diff
@@ -0,0 +1,12 @@
+@use '../../../../styles/utils';
+
+@include utils.global-styles() {
+  mat-radio-button {
+    --mdc-radio-selected-pressed-icon-color: var(--mat-sys-on-tertiary-fixed);
```

Comment:
The hover/pressed color for non-checked buttons is incorrect. Please update.
Use `:checked` css selector to differentiate between the two